app/src/main/res/layout/profile_chooser_profile_view.xml

```diff
@@ -25,7 +25,7 @@
     android:layout_height="wrap_content"
     android:gravity="center_horizontal"
     android:orientation="vertical"
-    app:layoutMarginBottom="@{hasProfileEverBeenAddedValue ? @dimen/space_20dp : @dimen/space_0dp}">
+    app:layoutMarginBottom="@{hasProfileEverBeenAddedValue ? @dimen/profile_already_added_margin : @dimen/space_0dp}">
```

Comment:
This case is a bit strange since it's conditional, but I think this name is pretty close to what we want. It seems to be missing what the actual thing having the margin is, though. Perhaps: ``profile_view_already_added_margin`` and ``profile_view_not_added_margin``. WDYT?

Reply:
The names you suggested describe its use better. Will change them accordingly.

Reply:
Resolving this on behalf of @BenHenning